kitsune/sumo/middleware.py

```diff
@@ -278,3 +277,30 @@ def process_request(self, request):
             response = HttpResponseRateLimited()
             patch_vary_headers(response, ['User-Agent'])
             return response
+
+
+class InAAQMiddleware(MiddlewareMixin):
+    """
+    Middleware that updates session's keys based on the view used.
+
+    aaq_* views -> in-aaq = True
+    """
+
+    def process_view(self, request, callback, callback_args, callback_kwargs):
+        # This key is used both in templates and the
+        # LogoutDeactivateUsersMiddleware
+
+        if not request.session or callback:
```

Comment:
I assume this is a typo, fixed here: https://github.com/mozilla/kitsune/pull/4371/files#diff-6f05e342c77cc4322987a6ff3c3edd64R293